### PR TITLE
AddressInput - Fixing text alignment

### PR DESCRIPTION
**Pull request by giladartzi**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
View storybook at: https://wix-wix-ui-icons-common-pr-470.surge.sh

**Comment by WixBuildServer** [comment]
View storybook at: https://wix-wix-ui-core-pr-470.surge.sh

File: packages/wix-ui-core/src/components/AddressInput/AddressInput.st.css [hunk 1]
```diff
@@ -19,4 +19,8 @@
 
 .icon {
 
-}
\ No newline at end of file
+}
+
+.optionContent {
+    flex: 1;
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/AddressInput/AddressInput.st.css]: Not sure why the core needs to decide this. It doesn't feel like a mistake, but I am not fully convinced as well.

Comment (giladartzi) [reply packages/wix-ui-core/src/components/AddressInput/AddressInput.st.css]: Basically, it causes the option to take the full width of its container. I think it's the expected behaviour for the core component

**Pull request merged**